bower.json

```diff
@@ -76,7 +76,8 @@
     "flair-visualizations": "master",
     "datatables.net-dt": "^3.2.2",
     "underscore": "^1.10.2",
-    "sql-formatter": "^1.1.2"
+    "sql-formatter": "^1.1.2",
```

Comment:
there is no change why to update this file...just checkout this file

Reply:
I have added angularjs-dropdown-multiselect package that's why its added comma